positron/modules/atom_browser_web_contents.js

```diff
@@ -29,6 +32,57 @@ let WebContents_prototype = {
   loadURL: function(url) {
     this._browserWindow._domWindow.location = url;
   },
+
+  openDevTools: function() {
```

Comment:
Yep, I can switch! I use those for new files but here I was matching the style of the other methods. Anyway, I can change.